### ISSUE TITLE
How to solve getGherkinRows() is undefined for the type DataTable

### ISSUE THREAD

**Issue by sanjay058** [rest-assured/rest-assured#1062]
DataTable table;
for(DataTableRow row : table.getGherkinRows()) - Compiler error :getGherkinRows() is undefined for the type DataTable

dependency : cucumbe-java 1.2.2

**Comment by johanhaleby** [comment]
What has this to do with REST Assured?

**Comment by sanjay058** [comment]
Yes, it is with Cucumber+ Rest assured, And project type is a maven. Here I m trying to read data from feature using datatable but getting above error.

Please suggest me how to solve it

**Comment by suruchi10** [comment]
I am also facing the same issue. I am using cucumber + rest assured with maven
